labs/sysdev-toolchain/sysdev-toolchain.tex

```diff
@@ -186,6 +186,12 @@ \section{Produce the toolchain}
 
 And wait!
 
+If the error message \code{Don't set LD_LIBRARY_PATH. It screws up the build.} appears, try to:
+\begin{bashinput}
+$ unset LD_LIBRARY_PATH
+\end{bashinput}
+then retry the build.
```

Comment:
I'm not favorable of this change. Indeed, this was very specific to your setup. In a normal setup, LD_LIBRARY_PATH is not set. I don't want to clutter the instructions with all possible issues that can arise on random setups, otherwise the instructions will be many pages of irrelevant details.

Reply:
Agreed.